### ISSUE TITLE
Create Model

### ISSUE THREAD

**Issue by vicentebolea** [vicentebolea/ml-bio-project#4]

**Comment by vicentebolea** [comment]
@olzhabay @badyalgaurav 

Let's use this thread to talk about the model:

Here is the accuracy of the 3 best classifier I used:

1. CNN (KERAS) : 0.39%
2. Random Forest: 0.48%
3. Gradient Boosting: 0.52% (I havent tried to tune it more)

I think we have to focus on the 3rd one.

Pull latest master commit.

To tune it, modify its parameters (n_stimators and learning_rate the most):
```
clf_rf = GradientBoostingClassifier(n_estimators=100, learning_rate=0.1, verbose=True)   
```
To run it:
````
 $ pip install --user pipenv
 $ pipenv install --skip-lock
 $ pipenv run python ./gradient_classifier.py
````

**Comment by vicentebolea** [comment]
Frequency analysis of input
```
   Cancer_Type         Tumor_Sample_ID     Gene_Name        Chromosome    
 BRCA    : 68115   TCGA-AP-A0LM: 10859   TTN    :  1656   1      : 37550  
 COADREAD: 63783   TCGA-AG-A002:  9863   TP53   :   977   2      : 27405  
 GBM     : 16559   TCGA-AP-A059:  9544   PIK3CA :   645   19     : 21812  
 LUAD    : 56108   TCGA-B5-A0JY:  9329   MUC16  :   612   3      : 21102  
 OV      : 15284   TCGA-B5-A11E:  8531   PTEN   :   403   11     : 21053  
 UCEC    :140714   TCGA-BS-A0UV:  8133   RYR2   :   403   12     : 19389  
                   (Other)     :304304   (Other):355867   (Other):212252  
 Start_Position       End_Position       Variant_Type Reference_Allele
 Min.   :     3380   Min.   :     3380   DEL: 11046   C      :137281  
 1st Qu.: 33760257   1st Qu.: 33760257   DNP:  1614   G      :136318  
 Median : 65168222   Median : 65168222   INS:  6085   T      : 37184  
 Mean   : 79625115   Mean   : 79625115   SNP:341802   A      : 37177  
 3rd Qu.:118948478   3rd Qu.:118948478   TNP:    16   -      :  6085  
 Max.   :249212544   Max.   :249212544                GG     :   518  
                                                      (Other):  6000  
  Tumor_Allele   
 A      :128483  
 T      :128480  
 G      : 44950  
 C      : 44878  
 -      : 10878  
 AA     :   502  
 (Other):  2392  

```

**Comment by badyalgaurav** [comment]
Btw you are not using nn now right? @vicenete??

**Comment by vicentebolea** [comment]
Yes, I stopped as Gradient booster yield better throughtput.

**Comment by olzhabay** [comment]
anybody can write about gradient boost? have no idea what to write about it :|
